src/app.hooks.js

```diff
@@ -118,6 +119,33 @@ const displayInternRequests = (level) => (context) => {
 	return context;
 };
 
+/**
+ * For errors without error code create server error with code 500.
+ * In production mode remove error stack and data.
+ * @param {context} context
+ */
+const errorHandler = (context) => {
+	if (context.error) {
+		if (context.error.hook) {
+			delete context.error.hook;
+		}
+		// statusCode is return by extern services / or mocks that use express res.status(myCodeNumber)
+		if (!context.error.code && !context.error.statusCode) {
+			context.error = new GeneralError(context.error.message || 'server error', context.error.stack || '');
+		}
+
+		// for error handling in sentrys and logs remove this keys
+		if (context.data) {
+			delete context.data.createdBy;
```

Comment:
we need this in the audit log, we can just filter these properties out in the project configuration within of sentry

Reply:
Ok, first step i remove this code and do not add in sentry. If we later want a filter we can add it to this time.

Reply:
done